.github/workflows/rollup-config-diff.yml

```diff
@@ -33,7 +33,48 @@ jobs:
 
     - name: compute diff
       working-directory: validation
-      id: run-command
+      id: run_command
       run: |
         ./scripts/diff-configs.sh ./generate-rollup-config/output-base ./generate-rollup-config/output-HEAD
+      continue-on-error: true
 
+    - name: Check command status and comment on PR
+      if: steps.run_command.outcome == 'failure'
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
```

Comment:
this looks like the same script as above. is there a way to deduplicate?

Reply:
Yes, I made a reusable action to deduplicate the code [3d49b7f](https://github.com/ethereum-optimism/superchain-registry/pull/447/commits/3d49b7f8c232c0c07820d2bb1e77662eddf1ad49)

Reply:
nice